crates/bevy_reflect/src/impls/std.rs

```diff
@@ -87,7 +87,8 @@ impl_reflect_value!(isize(
 ));
 impl_reflect_value!(f32(Debug, PartialEq, Serialize, Deserialize, Default));
 impl_reflect_value!(f64(Debug, PartialEq, Serialize, Deserialize, Default));
-impl_reflect_value!(String(
+impl_type_path!(str);
```

Comment:
Interesting, why is this needed?

Reply:
for `&'static str` and `Cow<'static, str>` since they have blanket implementations of `TypePath` with `?Sized` bounds.